### DISCUSSION TITLE
Password problem

### DISCUSSION THREAD

**Discussion by CISO-GYM** [wazuh/wazuh#26963] [Q&A]
Hello, when i want to do the "Changing the passwords in a distributed environment"
after i generated these passwords:

22/11/2024 13:35:00 INFO: The password for user admin is vOFUG2VDTH41?zL7+UvtYVWWj+ozNyye
22/11/2024 13:35:00 INFO: The password for user anomalyadmin is bO7IaEvSqwsOJ.9Nvtyi0G2Up.yPk?2j
22/11/2024 13:35:00 INFO: The password for user kibanaserver is YtKEoh9otLiYP8*2*RoYcxGSUZj40LF*
22/11/2024 13:35:00 INFO: The password for user kibanaro is dvZlbDiR1J1+9+YhwCXdOjqNLrP8VMq0
22/11/2024 13:35:00 INFO: The password for user logstash is 4X8r6ORC1OSkSpJznahJpIAkGVJ*mhnR
22/11/2024 13:35:00 INFO: The password for user readall is 5KUV?qglw.W.6hLF9Z+XxypHXuOrqR7u
22/11/2024 13:35:00 INFO: The password for user snapshotrestore is hHD.a7flZlquAA.45udtr47CT38x1NN0
WARNING: Wazuh indexer passwords changed. Remember to update the password in the Wazuh dashboard, Wazuh server, and Filebeat nodes if necessary, and restart the services.

I use bash wazuh-passwords-tool.sh --api --change-all --admin-user wazuh --admin-password <WAZUH_PASSWORD>
but i dont know which user and password to use,
I tried many scenarios:

1-
root@xxx:/home/xxx/wazuh# bash wazuh-passwords-tool.sh --api --change-all --admin-user wazuh --admin-password vOFUG2VDTH41?zL7+UvtYVWWj+ozNyye
22/11/2024 13:50:50 INFO: Updating the internal users.
22/11/2024 13:50:53 INFO: A backup of the internal users has been saved in the /etc/wazuh-indexer/internalusers-backup folder.
22/11/2024 13:50:53 ERROR: Invalid admin user credentials

2-
root@xxx:/home/xxx/wazuh# bash wazuh-passwords-tool.sh --api --change-all --admin-user admin --admin-password vOFUG2VDTH41?zL7+UvtYVWWj+ozNyye
22/11/2024 13:50:58 INFO: Updating the internal users.
22/11/2024 13:51:01 INFO: A backup of the internal users has been saved in the /etc/wazuh-indexer/internalusers-backup folder.
22/11/2024 13:51:01 ERROR: Invalid admin user credentials

**Comment by MiguelazoDS** [comment]
Hi @IRobinHood, 

Sorry for the delayed response, if you're using this password `vOFUG2VDTH41?zL7+UvtYVWWj+ozNyye` the user is `admin` not `wazuh`. 

More info here 

https://documentation.wazuh.com/current/user-manual/user-administration/password-management.html

**Comment by Damian-Mangold** [comment]
I closed it because it had been inactive for more than 7 days. Please reopen it if necessary.
